doc/api/debugger.md

```diff
@@ -202,3 +202,35 @@ To start debugging, open the following URL in Chrome:
 ```
 
 [TCP-based protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+
+## Trace Event Support for Node.js
+
+Trace Event provides a mechanism to centralize tracing information generated by
+V8, Node core, and userspace code.
+
+Tracing can be enabled by passing the `--enable-tracing` flag when starting a
+Node.js application.
+
+Tracing can be configured by additionally specifying a configuration file using the
+`--trace-config` flag:
+
+```txt
+node --enable-trace --trace-config=trace-config.json server.js
+```
+
+where `trace-config.json` may override any of the default configuration values:
```

Comment:
Configuration has been simplified to just specify enabled categories through a command line flag. Other configuration options are not useful at this time (only one `record_mode` supported by the buffer, systrace not being used by node, etc). We are working on getting a list of categories compiled here: https://github.com/v8/v8/wiki/Tracing%20V8. Currently node does not introduce any categories of its own. The `node` category introduced by this PR is just a placeholder and can be updated when the community decides how/where trace points should be added to core.